getting-started.md

```diff
@@ -179,9 +179,9 @@ sure that the rekor entry is created for it.
 kubectl -n ctlog-system get secrets ctlog-public-key -oyaml | sed 's/namespace: .*/namespace: default/' | kubectl apply -f -
 ```
 
-2) Create the two test jobs (checktree and check-oidc)  using this yaml (this may take a bit (~couple of mintues), since the two jobs are launched simultaneously)
+2) Create the two test jobs (checktree and check-oidc)  using this yaml (this may take a bit (~couple of minutees), since the two jobs are launched simultaneously)
```

Comment:
nit typo: `minutees` > `minutes`

Reply:
lol, hilarious 😆  I go fix a typo and introduce a new one. Strong work by yours truly 😆